api/src/LinkyLink/Models/LinkBundle.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
 
 namespace LinkyLink.Models
 {
     /// <summary>
     /// Data model class to represent a Link Bundle.
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
same as above

Reply:
I dont think there is a comment needed in the code for data model classes.